### ISSUE TITLE
Support for running info without a url

### ISSUE THREAD

**Issue by Lyeeedar** [flyway/flyway#2804]
It would be useful to be able to query for the migrations found locally without a url, to help with figuring out if the setup you have on disk is valid and contains what you expect it to contain.

**Comment by nyctef** [comment]
The current workaround (thanks Phil!) is to specify a disposable in-memory database as the flyway url (eg `jdbc:h2:mem:flyway_db`). This seems to work pretty well. This could potentially be implemented the same way in flyway itself with a null-object `Database` implementation which just returns empty results? H2 appears to add minimal overhead though so this probably isn't a huge deal.

One issue we ran into was doing the above caused all the migrations to show as "Pending", which is correct as far as flyway is concerned but caused some confusing behavior in RCC. We've worked around this for now on our end, but if this was implemented in flyway it might be worth thinking about what the migration status should show as.
